tools/pin_github_action.py

```diff
@@ -1,45 +1,30 @@
 from __future__ import annotations
 
 import argparse
-import json
-import os
 import re
-import urllib.request
+import subprocess
 from functools import lru_cache
 from typing import Sequence
-from urllib.error import HTTPError
 
 
 @lru_cache(maxsize=None)
-def get_sha(repo: str, ref: str, github_token: str) -> str:
+def get_sha(repo: str, ref: str) -> str:
```

Comment:
I see you're removing the usage of `github_token`; is it because `ls-remote` solves the problem?

Reply:
yeah, github actions can only be public repos -- so this is simpler and requires fewer creds